src/components/dropdownSearch/MultiSelectDropdownSearch/tests/SelectedOption.spec.tsx

```diff
@@ -32,10 +32,10 @@ describe('SelectedOption', () => {
 
     describe('render', () => {
       it('should render the display value in the selectedOption', () => {
-        const value: string = 'displayTest';
-        selectedOption.setProps({ value });
+        const label: string = 'displayTest';
+        selectedOption.setProps({label });
```

Comment:
you have a trailling space after `label`

Reply:
it's the linter i think

Reply:
Fixed